docs/step-attributes.md

```diff
@@ -31,3 +32,33 @@ Here are commonly seen use cases that need to be handled carefully, accounting R
 | Given Coffee costs $5.00 today | Given(@"Coffee costs \\$([\d\\.]+) today") | decimal | 5.00 |
 | Given Coffee costs $5.00 today | Given(@"Coffee costs $([\d\\.]+) today") | any type | error: will not match dollar sign |
 | Given Coffee costs $5.00 today | Given(@"Coffee costs ([\d\\.]+) today") | any type | error: will not match dollar sign |
+
+## BDD Notation
```

Comment:
Let's name it not BDD notation but instead "Special characters within scenario steps" or something along those lines. Developers who are new to BDD may be confused by the general terms.